internal/testcli/runner.go

```diff
@@ -0,0 +1,315 @@
+package testcli
```

Comment:
Just to double check, this one was just moved, no other code changes?

Reply:
Yes, moved and renamed a few functions. No functional changes.